contracts/vault/src/common.rs

```diff
@@ -0,0 +1,51 @@
+use crate::{prelude::*, state};
+#[allow(dead_code)]
+/// Calculates the total assets of the vault (balance + allocations)
+///
+/// # Parameters
+/// - `deps`: Dependencies for storage access
+/// - `env`: Contract environment
+///
+/// # Returns
+/// - `StdResult<Uint128>`: Sum of USDC balance and market allocations
```

Comment:
Note that may be irrelevant by the time I finish reviewing the code: market allocations can change in value based on changes in the LP token price. If I put 1000 USDC into an LP position, it may end up either more or less depending on trader profits and losses.